projects/llm-semconv.md

```diff
@@ -0,0 +1,72 @@
+# LLM Observability Semantic Conventions
+
+## Description
+
+This project aims to establish observability standards for Large Language Model (LLM) applications, focusing on API interactions and multi-step LLM frameworks. Key areas include:
+
+- Developing semantic conventions for standard and complex LLM interactions.
+- Initiating vendor-specific conventions as a foundation for broader contributions.
+- Concentrating on attributes relevant to Metrics and Spans.
+
+For Spans: Instrumentation and debugging guidance for multi-step LLM applications, with recommendations on user privacy and intellectual property security, marking sensitive attributes and events as opt-in.
```

Comment:
I think this:

> marking sensitive attributes and events as opt-in

Will continue to be a point of discussion, both on the opt-in/out nature and where it's represented. I believe in my initial PR the consensus was that Logs were the best signal for holding critical debugging information like the full prompt and LLM response.

Reply:
Agreed. This is somewhere we'll need much more input and be willing to evolve as we learn more and prototype around the conventions. I like the Events/Logs approach, but I only know the scenarios I'm directly involved with.